loan_modified.R

```diff
@@ -2,14 +2,17 @@
 library(ggplot2)
 
 #Load Banking data
-loan <- read.csv("data/loan.csv")
+loan <- read.csv("loan 2.csv")

```

Comment:
I have renamed file for myself for now, we can update it to final one. Please update it for yourself.